ignore.d.server/repo-dovecot

```diff
@@ -8,7 +8,10 @@
 ^\w{3} [ :[:digit:]]{11} [._[:alnum:]-]+ dovecot: (imap|IMAP)\([-_.@[:alnum:]]+\): Disconnected(: Logged out| for inactivity|: Disconnected|: Disconnected in [[:upper:]]+|: Too many invalid IMAP commands\.)?( in=[[:digit:]]+ out=[[:digit:]]+)?$
 ^\w{3} [ :[:digit:]]{11} [._[:alnum:]-]+ dovecot: (pop3|imap)-login: (Disconnected|Aborted login)(: Inactivity( during authentication)?|: Too many invalid commands)? (\((no auth attempts in [[:digit:]]+ secs|tried to use disallowed plaintext auth|disconnected before greeting, waited [[:digit:]]+ secs)?\):( user=<[-_.@[:alnum:]]*>,)?|\((auth failed, [[:digit:]]+ attempts( in [[:digit:]]+ secs)?|client didn't finish SASL auth, waited [[:digit:]]+ secs)\): user=<[^>]*>, method=PLAIN,|\(aborted authentication\): method=PLAIN,) rip=[.:[:xdigit:]]+, lip=[.:[:xdigit:]]+(, secured)?(, (TLS|SSL)( handshaking)?(: Disconnected)?)?(: SSL_(accept|read)\(\) syscall failed: Connection reset by peer|: SSL_(accept|read)\(\) failed: error:[[:xdigit:]]{8}:[^,]+)?(, session=<[[:alnum:]/+]+>)?$
 ^\w{3} [ :[:digit:]]{11} [._[:alnum:]-]+ dovecot: (imap|IMAP)\([-_.@[:alnum:]]+\): Connection closed(: Connection reset by peer)?( in=[[:digit:]]+ out=[[:digit:]]+)?$
-^\w{3} [ :[:digit:]]{11} [._[:alnum:]-]+ dovecot: (imap|IMAP)\([-_.@[:alnum:]]+\): Connection closed \(IDLE running for [[:digit:].]+ \+ waiting input for [[:digit:].]+ secs, ([[:digit:].]+ in locks, )?[[:digit:]]+ B in \+ [[:digit:]]+\+[[:digit:]]+ B out, state=wait-input\)( in=[[:digit:]]+ out=[[:digit:]]+)?$
+^\w{3} [ :[:digit:]]{11} [._[:alnum:]-]+ dovecot: (imap|IMAP)\([-_.@[:alnum:]]+\): Connection closed(|: read\(size=[[:digit:]]+\) failed: Connecttion reset by peer) \(IDLE running for [[:digit:].]+ \+ waiting input for [[:digit:].]+ secs, ([[:digit:].]+ in locks, )?[[:digit:]]+ B in \+ [[:digit:]]+\+[[:digit:]]+ B out, state=wait-input\)( in=[[:digit:]]+ out=[[:digit:]]+)?$
+
+#May 29 11:44:40 maggie dovecot: imap(michael.essbach@casa-due-pur.de): Connection closed: read(size=1231) failed: Connection reset by peer (IDLE running for 0.001 + waiting input for 0.001 secs, 2 B in + 10+0 B out, state=wait-input)    in=34340 out=14365865
```

Comment:
please remove the example from the rule file, preferably via rebase so no private email addresses are exposed here

(I've been thinking about including sample logs to CI-testing, but have not yet had time to build it)